packages/@okta/vuepress-site/docs/guides/configure-access-policy/configure-token-lifetime-per-client/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Configure a custom token lifetime per client
+---
+
+Use the following steps to configure an access token lifetime for a specific client.
+
+## Create the policy container
+
+1. In the Developer Console, navigate to **API > Authorization Servers**.
+
+2. Select the name of the authorization server that you want to apply the policy to.
+
+3. Select **Access Policies**, and then **Add Policy**.
+
+4. Enter a **Name** and a **Description** for the policy.
+
+5. Select **The following clients:** and start typing the names of the Okta OpenID Connect applications that you want to cover with the access policy. This field automatically displays a list of applications that match what you type.
+
+6. Click **Create Policy**.
+
+## Create the rule
```

Comment:
I might add some transitory text here to help explain why we need to make a rule, something like: "Now that you have created a policy container, the next step is to create one or more rules for that policy"